app/assets/javascripts/components/coordinates-search-form.js

```diff
@@ -0,0 +1,65 @@
+(function(window, $) {
+  'use strict';
+
+  var disabledButtonClassName = 'usa-button-disabled',
+      errorMessageHTML = '<div class="usa-alert usa-alert-error" role="alert"><div class="usa-alert-body"><p class="usa-alert-text">There was a problem determining your location. Please reload the page and try again.</p></div></div>';
+
+  var CoordinatesSearchForm = function(options) {
+    this.$container = options.$container;
+    this.$input = options.$input;
+    this.$button = options.$button;
+    this.$errorContainer = options.$errorContainer;
+    this.$noticeContainer = options.$noticeContainer;
+
+    if (navigator.geolocation) {
+      this.setup();
+    }
+  };
+
+  CoordinatesSearchForm.prototype = {
+    events: {
+      click: function(event) {
+        event.preventDefault();
+
+        this.$button.attr('disabled', true).addClass(disabledButtonClassName);
+
+        navigator.geolocation.getCurrentPosition(this.events.success.bind(this), this.events.error.bind(this), {
+          enableHighAccuracy: true,
+          maximumAge: 0,
+          timeout: 10000
+        });
+      },
+
+      error: function(error) {
+        this.$errorContainer.html(errorMessageHTML);
```

Comment:
Am I reading it correctly that you're not looking at the error code to present a user-friendly error message here?

Reply:
@jamesatheyDDS Correct. The error messages I observed (from a [PositionError](https://developer.mozilla.org/en-US/docs/Web/API/PositionError)) aren't terribly useful relative to what we want to convey to the user.